composer.json

```diff
@@ -32,6 +32,7 @@
         "php": "^7.4 || ^8.0",
         "doctrine/cache": "^1.11 || ^2.0",
         "doctrine/dbal": "^3.7.0 || ^4.0",
+        "doctrine/orm": "^2.17 || ^3.0",
```

Comment:
I don't think you should move this dependency here. When `$this->container->getParameter('doctrine.orm.auto_generate_proxy_classes')` is `true`, then it must mean `doctrine/orm` is installed, right?

Reply:
Oh, ofcourse, you're entirely correct. Moving this change back to require-dev, I do still think the version should be bumped, to avoid installing a version of ORM without the new class